src/test/kotlin/no/nav/tilleggsstonader/sak/brev/JournalførVedtaksbrevTaskTest.kt

```diff
@@ -91,4 +96,58 @@ class JournalførVedtaksbrevTaskTest {
         assertThat(taskSlot.captured.payload).isEqualTo(saksbehandling.id.toString())
         assertThat(taskSlot.captured.type).isEqualTo(DistribuerVedtaksbrevTask.TYPE)
     }
+
+    @Test
+    internal fun `dersom det finnes én brevmottaker skal opprettJournalpost og brevmottakerRepository-update kjøres én gang hver`() {
+        val brevmottaker = mockk<Brevmottaker>()
+        every { journalpostService.opprettJournalpost(any()) } returns ArkiverDokumentResponse("journalpostID", true, null)
+        every { brevmottakerRepository.update(any()) } returns brevmottaker
+
+        journalførVedtaksbrevTask.doTask(task)
+
+        verify(exactly = 1) { journalpostService.opprettJournalpost(any()) }
+        verify(exactly = 1) { brevmottakerRepository.update(any()) }
+    }
+
+    @Test
+    internal fun `derom det finnes to brevmottakere skal opprettJournalpost og brevmottakerRepository-update kjøres to ganger hver`() {
```

Comment:
Hadde også vært fint med en test der journalpostId er satt for mottaker 1, men ikke mottaker 2. Og så verifisere at man kun forsøker å journalføre den som ikke allerede er det.

Reply:
Fixed i siste commit